honeybee_energy/ventcool/opening.py

```diff
@@ -34,25 +34,59 @@ class VentilationOpening(object):
             cross ventilation will be induced. If False, the assumption is that
             the operable area is primarily on one side of the Room and there is
             no wind-driven ventilation. (Default: False)
+        air_mass_flow_coefficient_closed: A number in kg/s-m, at 1 Pa per meter of crack
+            length, used to calculate the mass flow rate when the opening is closed;
+            required to run an AirflowNetwork simulation. Some common values for this
```

Comment:
Change this from "required to run an AirflowNetwork simulation" to "only used in AirflowNetwork simulation."  The default should be 0 if it's not specified.